index.d.ts

```diff
@@ -1,99 +1,99 @@
-import { Url } from 'url';
+import { Url } from "url";
```

Comment:
my only nitpick on this is to ask that this file also follow the conventions we've set in the repo and use single quote. after that, I think we're good to go.

Reply:
May I add a .prettierrc file? The vscode prettier did that on save.